iOS_SDK/OneSignalSDK/Source/UNUserNotificationCenter+OneSignal.m

```diff
@@ -271,12 +271,11 @@ + (void) processiOS10Open:(UNNotificationResponse*)response {
     if (![OneSignalHelper isOneSignalPayload:response.notification.request.content.userInfo])
         return;
     
-    let isActive = [UIApplication sharedApplication].applicationState == UIApplicationStateActive &&
-                    OneSignal.notificationDisplayType != OSNotificationDisplayTypeNotification;
+    let isActive = [UIApplication sharedApplication].applicationState == UIApplicationStateActive;
     
     let userInfo = [OneSignalHelper formatApsPayloadIntoStandard:response.notification.request.content.userInfo
                                                       identifier:response.actionIdentifier];
-    let isAppForeground = [[UIApplication sharedApplication] applicationState] == UIApplicationStateActive;
+    let isAppForeground = [[UIApplication sharedApplication] applicationState] != UIApplicationStateBackground;
```

Comment:
Just curious, why can we do this change now?

Reply:
This was a latent bug that is good to change now since we are doing so much testing. the inactive state is still in the foreground